pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/TopicStats.java

```diff
@@ -313,6 +314,8 @@ public static void printTopicStats(PrometheusMetricStreams stream, TopicStats st
                     splitTopicAndPartitionIndexLabel);
             writeSubscriptionMetric(stream, "pulsar_subscription_filter_accepted_msg_count",
                     subsStats.filterAcceptedMsgCount, cluster, namespace, topic, sub, splitTopicAndPartitionIndexLabel);
+            writeSubscriptionMetric(stream, "pulsar_subscription_filter_estimated_msg_backlog",
```

Comment:
Are we technically estimating the `entry` backlog and not the `message` backlog?

Reply:
correct, but we use msg_backlog everywhere, so I prefer to have it consistent

Reply:
I haven't checked yet, but it seems like it might matter. We have an acceptance rate based on the message count, not entry count. Should we keep track of the msg/entry rate and then amplify the entry backlog? I will think more about this in a bit.